lib/realtime/server.ex

```diff
@@ -109,8 +126,11 @@ defmodule Realtime.Server do
         ) :: term()
   def update(update_term, server \\ __MODULE__)
 
-  def update({:vehicle_positions, vehicles_by_route_id, shuttles}, server) do
-    GenServer.cast(server, {:update, :vehicle_positions, vehicles_by_route_id, shuttles})
+  def update({:vehicle_positions, vehicles_by_route_id, shuttles, data_status}, server) do
```

Comment:
This seems like an annoying place to include the data status, as demonstrated by all the tests you needed to update to include this data. 

What do you think about having a separate `Server` for the current status, which itself can listen for for updates to vehicle positions and do the appropriate calculations?

Reply:
It is annoying. I gave it a shot in `Realtime.DataStatusRegistry` (though I left the calculations in the same place, and the new genserver just passes that result through)